pytorch_to_returnn/converter/converter.py

```diff
@@ -156,6 +157,27 @@ def _make_tf_feed_dict(self, input: Data):
     else:
       for i, size in input.size_placeholder.items():
         d[size] = [self._inputs_np.shape[input.get_batch_axis(i)]] * n_batch  # not so relevant
+    if out_entry:
+      visited_entries = set()
```

Comment:
I think I would directly collect the `non_deterministic_tensor_entries` right here in a list, and not recursively returning lists in the function and then doing `non_det_mods += ...`.
So you would add here:
```python
non_deterministic_tensor_entries =  []
```